src/app/main-content/header-music/music-control/music-control.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component } from '@angular/core';
+import { Song } from 'src/app/model/song.model';
+
+@Component({
+  selector: 'app-music-control',
+  templateUrl: './music-control.component.html',
+  styleUrls: ['./music-control.component.sass']
+})
+export class MusicControlComponent {
+  audio = new Audio();
+
+  testSong: Song = {
+    title: "Bohemian Rhapsody",
+    genre: "Rock",
+    releaseDate: "1975-10-31",
+    duration: 354, // Duration in seconds
+    songPath: "../../../../assets/bohemian_rhapsody.mp3"
+  };
+
+  currentSong: Song = this.testSong;
+
+  playSong(): void {
+    if (this.audio.paused) {
+      if (this.audio.readyState == 0) {
+        // this.songId = 0;
```

Comment:
delete unnecesary line comments

Reply:
it is just to remember where it should be updated

Reply:
ok